.ci/test_r_package_windows.ps1

```diff
@@ -153,10 +153,10 @@ if ($env:COMPILER -ne "MSVC") {
   if ($env:R_BUILD_TYPE -eq "cmake") {
     if ($env:TOOLCHAIN -eq "MINGW") {
       Write-Output "Telling R to use MinGW"
-      $env:BUILD_R_FLAGS = "c('--skip-install', '--use-mingw')"
+      $env:BUILD_R_FLAGS = "c('--skip-install', '--use-mingw', '-j4')"
     } elseif ($env:TOOLCHAIN -eq "MSYS") {
       Write-Output "Telling R to use MSYS"
-      $env:BUILD_R_FLAGS = "c('--skip-install', '--use-msys2')"
+      $env:BUILD_R_FLAGS = "c('--skip-install', '--use-msys2', '-j4')"
     } elseif ($env:TOOLCHAIN -eq "MSVC") {
       $env:BUILD_R_FLAGS = "'--skip-install'"
```

Comment:
I guess we are missing `-j` option here in case of fallback to MinGW/MSYS
https://github.com/microsoft/LightGBM/blob/b1facf50502fb51a1e60c05b3ec83f68289df497/R-package/src/install.libs.R#L200-L206

Reply:
I didn't include a `-j` in this CI script because if the jobs that are supposed to use MSVC end up falling back to MinGW/MSYS, they'll fail anyway:

https://github.com/microsoft/LightGBM/blob/b1facf50502fb51a1e60c05b3ec83f68289df497/.ci/test_r_package_windows.ps1#L215-L224

For users (not CI scripts), passing through `-j` options to MinGW/MSYS is handled in that part of `install.libs.R` that you linked to

https://github.com/microsoft/LightGBM/blob/3d0274caa8e188c75a7d77655b74f5cb8bdc3135/R-package/src/install.libs.R#L207

It looks like the link you shared is from a commit on `master`, not this PR.

Reply:
> I didn't include a `-j` in this CI script because if the jobs that are supposed to use MSVC end up falling back to MinGW/MSYS, they'll fail anyway:

Ah, completely forgot about that check! Thanks for the reminder.